src/main/java/com/google/api/codegen/transformer/py/ServiceViewModel.java

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.py;
+
+import com.google.api.client.repackaged.com.google.common.base.Preconditions; // ???
+import com.google.api.codegen.SnippetSetRunner;
+import com.google.api.codegen.config.ApiConfig;
+import com.google.api.codegen.gapic.CommonGapicCodePathMapper;
+import com.google.api.codegen.py.PythonGapicContext;
+import com.google.api.codegen.py.PythonImportHandler;
+import com.google.api.codegen.rendering.CommonSnippetSetRunner.TransitionViewModel;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.api.tools.framework.model.Interface;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+class ServiceViewModel implements ViewModel, TransitionViewModel {
```

Comment:
(As I was commenting with @lukesneeringer earlier, I don't even expect this class to survive till the end of the migration).